python/loops-lists/lists.py

```diff
@@ -0,0 +1,29 @@
+# Python Course
+# Lists
+
+# make a list of names:
+names = ["Joe", "Mary", "Eugene", "Katrina"]
+
+# print the whole list:
+print("here is a list of names", names)
```

Comment:
they don't know comma yet, so we might make a note somewhere to explain it quickly. 

wondering if it's easier to add strings here since they don't know functions and parameters yet (and we definitely don't want to get into `*args`!)